pkg/lib/journal.js

```diff
@@ -230,7 +234,34 @@
                 };
             if (count > 1)
                 parts['count'] = count;
-            if (prio < 4)
+            if (ident === 'abrt-notification') {
+                // Problem can be identified by 3 elements. Most likely the first, but in some
+                // cases other two can be used as well.
+                var all_p = [entry['PROBLEM_DIR'], entry['PROBLEM_DUPHASH'], entry['PROBLEM_UUID']];
+                var p;
+                var known = false;
+                for (var i = 0; i < all_p.length; i++) {
+                    if (all_p[i] in displayable_problems) {
+                        p = all_p[i];
+                        known = true;
+                        break;
+                    }
+                }
+                //Do not render unknown (deleted) problems
+                if (! known)
+                    return "";
```

Comment:
> So you think that if the same problem happens more than once, we should display first, second and all other occurrences?

Yes.  The journal is a time-line of events.  We shouldn't mess with that by omitting entries.  Let's say something bad happened on your server around 14:23 o'clock.  You go to the journal to see what has happened around that time.  If abrt has written something to the log around that time you want to see it.  It should not be omitted because the same thing has also happened later at 16:15.

> I cannot agree with you about the count though. I think it is quite an important information. You will definitely pay more attention if you see that something crashed several times.

The list of journal entries is not the place to show this kind of summary, I'd say.  We can collect statistics from the journal and show them in a separate "Notifications" UI, in a very prominent place. We some old ideas about this here: https://github.com/cockpit-project/cockpit/wiki/Notifications, maybe abrt is the thing that can drive this.